crates/flox/src/commands/mod.rs

```diff
@@ -169,10 +169,7 @@ impl FloxArgs {
 
         // Set the global Nix config via the environment variables in flox.default_args so that
         // subprocesses called by `flox` (e.g. `parser-util`) can inherit them.
-        let nix_backend: NixCommandLine = flox.nix(vec![]);
-        for (name, value) in nix_backend.defaults.environment.iter() {
-            std::env::set_var(name, value);
-        }
+        flox.nix::<NixCommandLine>(vec![]).export_config_env_vars();
```

Comment:
What's wrong with this?

Reply:
nothing against making it a method, all good.
Would just prefer that method on the `Flox` struct or at least within the repo.

Reply:
https://github.com/flox/runix/pull/44#discussion_r1288844416